docs/_data/toc.yml

```diff
@@ -269,6 +269,8 @@
             url: project/contributing/contributing-ui-notification-center
         - title: Docker Extension
           url: project/contributing/contributing-docker-extension
+        - title: Policy Engine
```

Comment:
```suggestion
        - title: Policies
```

Reply:
We’ll also want to move into alphabetical order.